### PR TITLE
Update favicon per use cross EnergySage verticals

### PR DESCRIPTION
**Pull request by tomleo**
### 🔗 Linked issue

- https://energysage.atlassian.net/browse/PLAT-245
- https://energysage.atlassian.net/browse/CORE-4160

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Got update favicons from Design
- Added a number of favicon formats & associated meta data for cross-devices.
- Hopefully this will be a reference for future nuxt verticals that wish to include the ES logo

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [ ] I have documented testing approach

### CODE REVIEW

File: es-design-system/static/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
+            <TileColor>#b95100</TileColor>
```
Comment (tomleo) [inline es-design-system/static/browserconfig.xml:6]: This color isn't in the design-system, but specific to the logo.

File: es-design-system/static/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
```
Comment (RyanMulready) [inline es-design-system/static/site.webmanifest:16]: Why not use `$primary` here?

**Review by RyanMulready — APPROVED** [review]
This seems like it would work fine. I might suggest looking at [nuxt/pwa](https://pwa.nuxtjs.org) for `nuxt` applications. It abstracts alot of this away for a cleaner implementation. CSM uses it and I like it a lot.

File: es-design-system/static/site.webmanifest (hunk 2 above)
Comment (tomleo) [reply es-design-system/static/site.webmanifest:16]: Will that work in the static-directory?

Comment (RyanMulready) [reply es-design-system/static/site.webmanifest:16]: No but you can hard code it.

Comment (tomleo) [reply es-design-system/static/site.webmanifest:16]: The white-background `#fff` was in consultation with Design. Similarly `#b95100` was recommended over the design-system orange color`#ffbf57` or it's associated gradients. But we can def revisit this.

I think the generated content from [realfavicongenerator](https://realfavicongenerator.net/) covers everything the *Icon Module* from `nuxt/pwa` would cover. But I wonder if the [static/site.webmanifest](https://github.com/EnergySage/es-ds/pull/328/files#diff-6f0ce75410a8767cc13157437c1c31ff65ba5ccbfa66d167ffc80645b7c9a573) would interfear with verticals using [Web App Manifest](https://pwa.nuxtjs.org/manifest) 🤔

**Pull request merged**